src/backend/mock-data.ts

```diff
@@ -0,0 +1,112 @@
+export const featureMock = {
```

Comment:
Er denne nødvendig? Ettersom toggles blir lagt til og fjernet regelmessig tenker jeg kanskje at det beste er å ikke ha noen toggler i denne testriggen. Kan diskuteres! :) 

Vi sender vel også inn alle togglene vi trenger, og isåfall kunne vi kanskje bare mappet over dem og satt alt til true?

Reply:
endret til å returnere true på alle togglene som sendes inn